pages/en/lb4/Defining-and-validating-the-API.md

```diff
@@ -302,20 +310,18 @@ import { ProductController, DealController, CategoryController } from "./control
 export class YourMicroservice extends RestApplication {
 
   constructor() {
-    super();
+    super({
+      rest: {
+        port: 3001
+      }
+    });
     const app = this;
 
     app.controller(ProductController);
     app.controller(DealController);
     app.controller(CategoryController);
-
-  }
-  async start() {
-    const server = await app.getServer(RestServer);
-    // inject your spec here!
-    server.api(spec);
-    server.bind("rest.port").to(3001);
-    await super.start();
```

Comment:
Don't we still need to call `super.start()`?

Reply:
@b-admike I think in this case it's enough to just inherit `start()` function from `server`. No need to override it.